packages/xstate-react/src/useActor.ts

```diff
@@ -28,7 +44,7 @@ export function useActor<TEvent extends EventObject, TEmitted = any>(
   useIsomorphicLayoutEffect(() => {
     actorRefRef.current = actorRef;
     setCurrent(getSnapshot(actorRef));
-    const subscription = actorRef.subscribe(setCurrent);
+    const subscription = actorRef.subscribe((emitted) => setCurrent(emitted));
```

Comment:
This is also a situation of how we don't utilize `error` and `complete` callbacks at all. What role do they play for us?

Reply:
We can/should use a full `observer` object for these. I'll make that change.